src/init/features/firestore/index.ts

```diff
@@ -1,10 +1,12 @@
+import { ensureLocationSet } from "../../../ensureCloudResourceLocation";
 import requireAccess = require("../../../requireAccess");
 import * as rules from "./rules";
 import * as indexes from "./indexes";
 
 export async function doSetup(setup: any, config: any): Promise<void> {
   setup.config.firestore = {};
 
+  ensureLocationSet(setup.projectLocation, "firestore");
```

Comment:
"Cloud Firestore" probably

Reply:
Reminds me of that bot that we have :) Changed.